projects/storefrontapp/src/index.html

```diff
@@ -79,7 +79,7 @@
     <script
       id="smartedit-injector"
       src="webApplicationInjector.js"
-      data-smartedit-allow-origin="localhost:9002"
+      data-smartedit-allow-origin="cds-1905-k8s-us.prod.argonauts.cd.ydev.hybris.com:9002"
```

Comment:
I understand that this is something that you need for development / testing, but this should not end up in `develop` branch. It would be good if you can document this in a `README.md` file placed in the root of the `cds`  lib. This will help you to just copy-paste value for development, and it will be useful for us as a documentation to better understand CDS.

Reply:
Reverted